jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -7,7 +7,7 @@
         - github:
             url: https://github.com/kubernetes/kubernetes
         - throttle:
-            max-total: 0  # Unlimited
+            max-total: '{max-total}'
```

Comment:
That's not very easy to do with JJB. It might be worth splitting these three jobs into their own files soon.